cmd/gnodev/test.go

```diff
@@ -80,13 +92,11 @@ func testApp(cmd *command.Command, args []string, iopts interface{}) error {
 			if verbose {
 				cmd.ErrPrintfln("=== PREC  %s", pkgPath)
 			}
-			pkgPathSafe := strings.ReplaceAll(pkgPath, "/", "~")
-			tempdir := filepath.Join(tempdirRoot, pkgPathSafe)
-			if err = os.MkdirAll(tempdir, 0o755); err != nil {
-				log.Fatal(err)
-			}
-			precompileOpts := precompileOptions{
-				Output: tempdir,
+			// pkgPathSafe := strings.ReplaceAll(pkgPath, "/", "~")
```

Comment:
Leftover comment?

Reply:
It was intentional. Actually, I wanted @moul's feedback on this. Why did he choose to do this?